lnrpc/switchrpc/switch.proto

```diff
@@ -117,3 +135,42 @@ enum ErrorCode {
     */
     INTERNAL = 8;
 }
+
+message TrackOnionRequest {
+    // The payment hash associated with the HTLC. This is needed for tracking
+    // and debugging purposes.
+    bytes payment_hash = 1;
+
+    // The attempt ID uniquely identifying this payment attempt. The caller can
+    // expect to track results for the payment via this attempt ID.
+    uint64 attempt_id = 2;
```

Comment:
let's keep the change in a side branch until we have better attempt id support for multiple routers

Reply:
Fully updating the Switch and rest of daemon (everywhere `attempt id` is referenced) could take some time though.